akka-docs/src/main/paradox/stream/stream-customize.md

```diff
@@ -181,9 +181,9 @@ and `abortReading()`
 
 An example of how this API simplifies a stage can be found below in the second version of the `Duplicator`.
 
-### Custom linear processing stages using GraphStage
+### Custom linear operators using GraphStage
 
-Graph stages allows for custom linear processing stages through letting them
+GraphStage allows for custom linear processing stages through letting them
 have one input and one output and using `FlowShape` as their shape.
```

Comment:
Here I converted "Graph stages" to "GraphStage".  
Actually does the sentence look little weird? and should be re-written like "To define custom linear operators, you should extend GraphStage using `FlowShape` which has one input and one output." ?

Reply:
yeah good suggestion, please reword and link to stream-customize from the GraphStage, let's  merge once that is done

Reply:
Sure, updated the sentence. stream-customize.md is this file, so I'm not linking from `GraphStage` here.